apps/typegpu-docs/src/content/examples/rendering/3d-fish/index.ts

```diff
@@ -50,10 +49,11 @@ const oceanFloorModel = await loadModel(
 
 // buffers
 
-const fishDataBuffers = Array.from({ length: 2 }, () =>
+const fishDataBuffers = Array.from({ length: 2 }, (idx) =>
```

Comment:
```suggestion
const fishDataBuffers = Array.from({ length: 2 }, (_, idx) =>
```

index is the second argument